### PR TITLE
Bradcrumbs Added

### PR DESCRIPTION
**Pull request by hammad-Ikhlaq-7vals**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -2,5 +2,6 @@
 class UsersController < ApplicationController
   def index
     @pagy, @users = pagy(User.where(params[:company_id]),  items: 5)
+    add_breadcrumb t('shared.users'), :company_users_path
```
Comment (hafsabatool7) [inline app/controllers/users_controller.rb]: Make it the first line of the method.
Add respond_to block
Add load_and_authorize
Add cancancan abilities

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 include Pagy::Backend
 class UsersController < ApplicationController
-  before_action :authenticate_user!
+  load_and_authorize_resource
   def index
+    add_breadcrumb t('shared.users'), :company_users_path
     @pagy, @users = pagy(User.where(params[:company_id]),  items: 5)
```
Comment (hafsabatool7) [inline app/controllers/users_controller.rb]:
```suggestion
    @pagy, @users = pagy(@users,  items: 5)
```
use the @users loaded by cancancan

File: app/views/layouts/_navbar.html.erb [hunk 3]
```diff
@@ -1,6 +1,38 @@
-<% if user_signed_in? %>
-  <%= render 'layouts/signed_in_user_navbar' %>
-<% else %>
-  <%= render 'layouts/normal_user_navbar' %>
-<% end %>
-<hr>
\ No newline at end of file
+<nav class='navbar navbar-expand-lg navbar-light bg-light'>
```
Comment (hafsabatool7) [inline app/views/layouts/_navbar.html.erb]: Does it now render 3 navbars? Verify.

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (hammad-Ikhlaq-7vals) [reply app/controllers/users_controller.rb]: done

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (hammad-Ikhlaq-7vals) [reply app/controllers/users_controller.rb]: done

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -2,10 +2,12 @@
 class UsersController < ApplicationController
   load_and_authorize_resource find_by: :sequence_num, through: :current_company
   def show
+    add_breadcrumb t('users.username', user_name: current_user.name), :user_path
```
Comment (hafsabatool7) [inline app/controllers/users_controller.rb]: current_user.name ? Current user can be different from the user on whose show page one is.

File: config/locales/en.yml [hunk 5]
```diff
@@ -134,6 +134,7 @@ en:
     name: 'name'
     role: 'role'
     company: 'company'
+    username: "%{user_name}"
```
Comment (hafsabatool7) [inline config/locales/en.yml]: No need to put it in internationalization if it does not contain any text. This is just outputting the value of the variable.

**Pull request merged**